app/assets/javascripts/auth_source_ldap.js

```diff
@@ -1,3 +1,25 @@
+function test_connection(item, url) {
+  $('#test_connection_indicator').show();
+  $(item).addClass("disable-link");
```

Comment:
I think the bootstrap button should support a "disabled" class already, which also changes the colour so it doesn't look like it's clickable.